packaging/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: skipper eskip docker-push
```

Comment:
technically, skipper and eskip are not PHONY. They both generate a binary and you could use make here to only build it when actually necessary